### ISSUE TITLE
Hyperparameter optimization giving abnormally high energy noise

### ISSUE THREAD

**Issue by MorAnyAn** [mir-group/flare#403]
Hi, FLARE developer team,

I have been using FLARE (ACE descriptor + sparse GP) lately.

The on-the-fly training was successful and I am glad that the resultant model has provided a decent force validation compared with DFT results.

However, the total potential energy given by the GP model is observing a constant shift, and the amount of such shifting is consistent across validation sets with different energy ranges:

![image](https://github.com/mir-group/flare/assets/53519671/845c6d1e-81ce-4827-ad6b-5f81b482af17)

In other attempts of training, such shift may be different as well:

![K3_Energy_Validation_FLARE_1](https://github.com/mir-group/flare/assets/53519671/3def89c5-8e46-4ce5-8757-8b419ffce831)

It seems to be a issue in hyperparameter optimization, since the hyperparameters for the noise of energy ended up relatively high:

![image](https://github.com/mir-group/flare/assets/53519671/3aae9791-d27a-456b-9bb6-d65b877e5f0a)
 
![image](https://github.com/mir-group/flare/assets/53519671/024f7e8e-6678-49f5-8db0-4e547df857ea)

As for the related setting, I have set the initial noise of energy at 0.063 as the recommended 0.001*atom numbers; hyperparameters are optimized after every 10 DFT calls and maximum iterations of optimization is 100.

Do you have any suggestion on this issue?

**Comment by jonpvandermause** [comment]
We are revisiting this against the current release. Could you provide a minimal current reproduction, including the input YAML, FLARE/ASE versions, representative training output, and the resulting model or hyperparameters? In particular, please include the energy labels and any per-species energy offsets used. That will let us distinguish a remaining optimizer issue from an energy-reference/configuration effect.
